testgrid/config.yaml

```diff
@@ -6301,6 +6311,15 @@ dashboards:
   - name: kops-gce-ha
     test_group_name: ci-kubernetes-e2e-kops-gce-ha
 
+# Gardener Dashboard
+- name: conformance-gardener
+  dashboard_tab:
+  - name: "v1.10 AWS"
```

Comment:
Quick question: what kubernetes build is this? Is this a continuous build from [the release-1.10 branch](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.10) or one of the tagged/released builds? (just for trying to keep the test tab names clear between what people are posting results for)

Reply:
Though we can update this later, it will be more relevant if we go to add this signal to the `conformance-all` dashboard.

Reply:
To my best knowledge it is related to the release-1.10 branch. However, I need to check it.

Reply:
Some clarification: Gardener uses the prebuild images e.g. on https://github.com/kubernetes/kubernetes/releases/tag/v1.10.4 - not the release branch as I mentioned before.

Reply:
Great, thanks!
We're still figuring out the best scheme to convey this, I think these are good for now. Perhaps in a follow up we can surface these in more dashboards, if we do I may tweak the description and / or tab names and CC you. 👍